src/main/java/com/epam/university/java/core/task005/PiHolderImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.university.java.core.task005;
+
+/**
+ * PiHolder implementation.
+ */
+public final class PiHolderImpl implements PiHolder {
+
+    /**
+     * numerator.
+     */
+    private int first;
+
+    /**
+     * denominator.
+     */
+    private int second;
+
+    PiHolderImpl(final int first, final int second) {
```

Comment:
Constructor should be public

Reply:
why?
we are not using this class outside of the package?
by the way, that's what IDEA tip tells me about it, and highlights the modificator.
that's annoying

Reply:
Okay